src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { keysToRemove, extraData, type } = action;
+
+    switch (true) {
+      case type === 'addProperties':
```

Comment:
same comment
[usefulLink](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch)
```suggestion
    switch (type) {
      case 'addProperties':
```
fix everywhere

Reply:
not fixed. If your have some problems with deploy fill free to ask in fe_chat